library/src/blas2/rocblas_ger_strided_batched.hpp

```diff
@@ -0,0 +1,109 @@
+/* ************************************************************************
+ * Copyright 2016-2019 Advanced Micro Devices, Inc.
+ * ************************************************************************ */
+#include "handle.h"
+#include "logging.h"
+#include "rocblas.h"
+#include "utility.h"
+
+template <typename T, typename U>
+__global__ void ger_strided_batched_kernel(rocblas_int m,
+                                           rocblas_int n,
+                                           U           alpha_device_host,
+                                           const T* const __restrict__ xa,
+                                           rocblas_int incx,
+                                           rocblas_int stridex,
+                                           const T* const __restrict__ ya,
+                                           rocblas_int incy,
+                                           rocblas_int stridey,
+                                           T* const    Aa,
+                                           rocblas_int lda,
+                                           rocblas_int strideA)
+{
+
+    ptrdiff_t tx = hipBlockIdx_x * hipBlockDim_x + hipThreadIdx_x;
+    ptrdiff_t ty = hipBlockIdx_y * hipBlockDim_y + hipThreadIdx_y;
+
+    if(tx < m && ty < n)
+    {
+        auto alpha = load_scalar(alpha_device_host);
+        T*   A;
+        const T* __restrict__ x;
+        const T* __restrict__ y;
+        A = Aa + hipBlockIdx_z * strideA;
+        x = xa + hipBlockIdx_z * stridex;
+        y = ya + hipBlockIdx_z * stridey;
+

```

Comment:
It should be:

A = Aa + hipBlockIdx_z * stride + shiftA;
x = xa + hipBlockIdx_z * stridex + shiftx;
y = ya + hipBlockIdx_z * stridey + shifty;